addon/appModules/notepad++/editWindow.py

```diff
@@ -126,7 +128,19 @@ def script_goToFirstOverflowingCharacter(self, gesture):
 	script_goToFirstOverflowingCharacter.category = "Notepad++"
 
 	def script_reportLineInfo(self, gesture):
-		ui.message(self.parent.next.next.firstChild.getChild(2).name) 
+		lineInfo = self.parent.next.next.firstChild.getChild(2).name
+		# Get only the numbers we want from the statusBar.
+		lineInfoExpression = re.compile(r"^Ln\D+(\d\w*)\D+(\d\w*)\D+(\d\w*)\D+(\d\w*)")
+		lines, columns, selectedCharacters, selectedLines = lineInfoExpression.match(lineInfo).groups()
+		#Translators: The line and column position of the cursor.
```

Comment:
The only issue with this is that some languages might want to choose a different order. I'm aiming to merge this addon into core, @JCsteh, do you think this will be an issue come that time?